.github/workflows/versioned-security-agent.yml

```diff
@@ -0,0 +1,54 @@
+# Daily workflow to run versioned tests with security agent enabled 
+# This also can be used to run versioned tests with a newer versioned of 
+# security agent since we pin it.
+
+name: Versioned Tests w/ Security Agent 
+
+on:
+  workflow_dispatch:
+    inputs:
+      mode:
+        description: Versioned test mode
+        type: choice
+        options:
+          - major
+          - minor
+        default: major
+        required: false
+      version:
+        type: string
+        description: Version of security agent to test
+        required: false
+  push:
```

Comment:
Prob want to remove push/pull_request here. I just had these while we were in the staging branch

Reply:
i'll wait to remove until approval in case we want these to run again

Reply:
maybe but this is diff as it accepts inputs on manual runs